### PR TITLE
[PROF-9342] Add issue with Ruby profiler dir class to profiler FAQ 

### PR DESCRIPTION
**Pull request by ivoanjo**
### What does this PR do? What is the motivation?

This PR adds one more issue we uncovered with the Ruby profiler to the profiler FAQ.

I've also gone ahead and cleaned up some of the references we had to older Ruby profiler versions. Given that 1.11.0 and 1.23.0 are > 1 year old, I think it makes more sense for the text to just assume customers are using 1.11.0 or later, rather than explaining how there's this "new" thing that changed in those versions.

### Merge instructions

This PR should only be merged after [version 2.2.0 of the Ruby library gets released](https://github.com/DataDog/dd-trace-rb/milestone/147). Aka please review, and I'll put a reminder and come back once the release is out :)

- [ ] Please merge after reviewing

### Additional notes

N/A

### CODE REVIEW

File: content/en/profiler/profiler_troubleshooting/ruby.md [hunk 1]
```diff
@@ -45,9 +45,9 @@ Datadog.configure do |c|
 end
 ```
 
-## Unexpected failures or errors from Ruby gems that use native extensions in `dd-trace-rb` 1.11.0+
+## Unexpected failures or errors from Ruby gems that use native extensions
 
-Starting from `dd-trace-rb` 1.11.0, the "CPU Profiling 2.0" profiler gathers data by sending `SIGPROF` unix signals to Ruby applications, enabling finer-grained data gathering.
+The Ruby profiler gathers data by sending `SIGPROF` unix signals to Ruby applications, enabling finer-grained data gathering.
```
Comment (kayayarai) [inline content/en/profiler/profiler_troubleshooting/ruby.md:50]:
```suggestion
The Ruby profiler gathers data by sending `SIGPROF` UNIX signals to Ruby applications, enabling finer-grained data gathering.
```

**Review by kayayarai — CHANGES_REQUESTED** [review]
Fixing a linter warning

**Review by kayayarai — APPROVED** [review]
Thanks Ivo, looks good. Ping us in #documentation when it's ready to merge

**Comment by ivoanjo** [comment]
Version 2.2.0 of the Ruby library has been released, please go ahead and merge this PR :)

**Comment by ivoanjo** [comment]
Thank you!

**Pull request merged**